README.md

```diff
@@ -202,16 +202,18 @@ Programmatic Errors indicating that a non-fatal error occurred, such as an appli
 
 #### Creating a Programmatic Error
 
-Programmatic errors are created by adding [span events](https://opentelemetry.io/docs/instrumentation/python/manual/#adding-events) with a custom attribute being set with the key name `lumigo.type`.
+Programmatic errors can now be created easily using the `create_programmatic_error` function provided by the `lumigo_opentelemetry` package. This allows you to capture and report errors with minimal setup, without requiring direct interaction with the OpenTelemetry trace SDK.
 
-For example, you could add a programmatic error as follows:
+For example, you can add a programmatic error as follows:
 
 ```python
-from opentelemetry.trace import get_current_span
+from lumigo_opentelemetry import create_programmatic_error
 
-get_current_span().add_event('<error-message>', {'lumigo.type': '<error-type>'})
+create_programmatic_error("Error message", "ErrorType")
```

Comment:
Can you write an actual message and a type? for example
The customer 123 was not found
CustomerNotExist

Reply:
Ok